docs/web.md

```diff
@@ -227,3 +228,30 @@ bootstrap()
 ```
 
 See the [nestjs-pino readme](https://npm.im/nestjs-pino) for more info.
+
+
+<a id="h3"></a>
+## Pino with H3
+
+```sh
+npm install pino-http
+```
+
+```js
+import { createServer } from 'http'
+import { createApp } from 'h3'
+import pino from 'pino-http'
+
+const app = createApp()
+
+app.use(pino())
+
+app.use('/', (req) => {
+  req.log.info('something')
+  return 'hello world'
+})
+
+createServer(app).listen(process.env.PORT || 3000)
+```
+
+See the [pino-http readme](https://npm.im/pino-http) for more info.
```

Comment:
```suggestion
See the [pino-http readme](https://npm.im/pino-http) for more information.
```

Reply:
Do we also need to change the rest of `for more info` to `for more information`?

Reply:
Ideally, yes. But that does not seem to be in scope of this PR.